content/automate/ManageIQ/Cloud/VM/Provisioning/StateMachines/VMProvision_VM.class/__methods__/retry_placement.rb

```diff
@@ -0,0 +1,29 @@
+#
+# Description: This method updates the provisioning status and retries
+# from the Placement step.
+# Required inputs: status
+#
+
+prov = $evm.root['miq_provision']
+unless prov
+  $evm.log(:error, "miq_provision object not provided")
+  exit(MIQ_STOP)
+end
+status = $evm.inputs['status']
```

Comment:
@mansam 
You would need some condition here to check that Placement really failed, if you had a script error this would keep running for ever.